<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_abir</class>
 <widget class="QMainWindow" name="gestion_abir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/hand-painted-watercolor-background-with-sky-clouds-shape_24972-1095.jpg);
background-image: url(:/new/prefix1/img/maxresdefault.jpg);
background-color: rgb(255, 170, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1051</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/img/maxresdefault.jpg);
background-color: rgb(230, 0, 0);</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Articles</string>
     </attribute>
     <widget class="QLineEdit" name="rech_article">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="table_article">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>71</y>
        <width>651</width>
        <height>331</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#d54e21;&quot;&gt;Liste des articles :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="article_sel_col">
      <property name="geometry">
       <rect>
        <x>591</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Titre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Categorie</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Contenu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Date arch</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Archive</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Archives</string>
     </attribute>
     <widget class="QTableView" name="table_arch">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>651</width>
        <height>361</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#d54e21;&quot;&gt;liste des archives :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rech_arch">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="arch_sel_col">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Intitule</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Droit Access</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Traitement</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Creation</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="export_excel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>430</y>
        <width>381</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Excel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/img/maxresdefault.jpg);
background-color: rgb(230, 0, 0);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionadd_article"/>
   <addaction name="actiondelete_article"/>
   <addaction name="separator"/>
   <addaction name="actionadd_archive"/>
   <addaction name="actiondelete_archive"/>
   <addaction name="separator"/>
   <addaction name="actionarchiver_article"/>
   <addaction name="separator"/>
   <addaction name="actionenvoyer_article"/>
   <addaction name="separator"/>
   <addaction name="actiondeconnecter"/>
  </widget>
  <action name="actionadd_article">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/img/Add-icon.png</normaloff>:/new/prefix1/img/Add-icon.png</iconset>
   </property>
   <property name="text">
    <string>add_article</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/ajouter article.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actiondelete_article">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/img/delete-icon.png</normaloff>:/new/prefix1/img/delete-icon.png</iconset>
   </property>
   <property name="text">
    <string>delete_article</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/supprimer article.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Backspace, Ctrl+D</string>
   </property>
  </action>
  <action name="actionadd_archive">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/img/images (2).jfif</normaloff>:/new/prefix1/img/images (2).jfif</iconset>
   </property>
   <property name="text">
    <string>add_archive</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actiondelete_archive">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/img/images (1).jfif</normaloff>:/new/prefix1/img/images (1).jfif</iconset>
   </property>
   <property name="text">
    <string>delete_archive</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionarchiver_article">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/img/archiver.jpg</normaloff>:/new/prefix1/img/archiver.jpg</iconset>
   </property>
   <property name="text">
    <string>archiver_article</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionenvoyer_article">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/img/backmain.jpg</normaloff>:/new/prefix1/img/backmain.jpg</iconset>
   </property>
   <property name="text">
    <string>envoyer_article</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actiondeconnecter">
   <property name="icon">
    <iconset>
     <normaloff>img/Windows-Log-Off-icon.png</normaloff>
     <normalon>:/new/prefix1/img/Windows-Log-Off-icon.png</normalon>img/Windows-Log-Off-icon.png</iconset>
   </property>
   <property name="text">
    <string>Se deconnecter</string>
   </property>
   <property name="toolTip">
    <string>deconnecter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
